docs/dataset/advanced-usage/extract-metadata-text.md

```diff
@@ -0,0 +1,274 @@
+---
+icon: kolena/comparison-16
```

Comment:
Let's pick out an icon for this page from https://blueprintjs.com/docs/#icons/icons-list and add both 16px and 20px icon svg files to /docs/overrides/.icons/kolena/

Reply:
I've added the citation one - seems relevant to text. 

<img width="459" alt="image" src="https://github.com/kolenaIO/kolena/assets/68381793/0d79d765-96fc-479f-9ac7-9dc6466d1c7a">


was also considering some others like search text, search signal

<img width="291" alt="image" src="https://github.com/kolenaIO/kolena/assets/68381793/fe02ccab-004d-406a-99ac-9a5c0eee1a3d">

Reply:
Yeah I think properties one is nice.